### ISSUE TITLE
Questions about SSL

### ISSUE THREAD

**Issue by develop-productivity** [soumik-kanad/diffssl#1]
Hello, thank you for putting up such a great job. 
I have a question, why do you call this work as a self-supervised learning method? It seems that this paper does not explain the specific training loss used. In this repo, cross-entropy loss and the target of the data set are used, so I think this does not meet the scope of self-supervised learning.

**Comment by soumik-kanad** [comment]
Hi,
Consider Masked AutoEncoders (MAE), which uses reconstruction pretraining which requires no labels. Similarly, we use a model trained to unconditionally generate images using diffusion process, which also does not use any labels. 
Now even for MAE, when you want to test its features on Imagenet classification, you need ImageNet labels+crossentropy loss to either train a linear layer (linear probing) or finetuning the whole backbone along with a linear layer. Similarly for our models. 

The diffusion process is the self-supervised pretraining in this work, like masked reconstruction is for MAE or contrastive learning is for SimCLR. For all of these methods the downstream task may require labels for training.

**Comment by soumik-kanad** [comment]
Hi,
Consider Masked AutoEncoders (MAE), which uses reconstruction pretraining which requires no labels. Similarly, we use a model trained to unconditionally generate images using diffusion process, which also does not use any labels.
Now even for MAE, when you want to test its features on Imagenet classification, you need ImageNet labels+crossentropy loss to either train a linear layer (linear probing) or finetuning the whole backbone along with a linear layer. Similarly for our models.

The diffusion process is the self-supervised pretraining in this work (we pick a pretrained ADM), like masked reconstruction is for MAE or contrastive learning is for SimCLR. For all of these methods the downstream task may require labels for training.
